.github/workflows/blui-ci.yml

```diff
@@ -0,0 +1,114 @@
+name: Build
+
+on:
+  push:
+    branches: ['dev', 'master']
+  pull_request:
+    branches: ['dev', 'master']
+  pull_request_target:
+    types:
+      - opened
+    branches:
+      - '*/*'
+
+permissions:
+  pull-requests: write
+  contents: read
+
+jobs:
+  prettier_lint:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [20.x]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'yarn'
+      - run: yarn
+      # - run: yarn docs-prettier
+      #   working-directory: docs
+    #   - run: yarn docs-lint
+    #     working-directory: docs
```

Comment:
We should be able to run lint and prettier right? We have the commands

Reply:
I saw your note in the PR description...can we log a bug for next sprint and assign it to @ArshdeepSingh3-eaton — this was supposed to be configured during the initial repository setup.